src/wp-admin/css/about.css

```diff
@@ -1,3 +1,4 @@
+/* @no-color-match */
```

Comment:
This is just for [the automated replacement tool](https://github.com/ryelle/wp-core-color-list/tree/main/postcss), it should be removed before this is merged.